### PR TITLE
Remove kubernetes 1.15 from UI selection

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
This should only stay here as an overwrite until we bump the chart version

### CODE REVIEW

File: addons/kommander/1.x/kommander.yaml [hunk 1]
```diff
@@ -74,3 +74,6 @@ spec:
           repository: mesosphere/kubeaddons-catalog
           tag: "v0.8.2"
           pullPolicy: IfNotPresent
+      
+      kommander-ui:
+        kubernetesVersionsSelection: '["1.16.4", "1.16.3"]'
```
Comment (shaneutt) [inline addons/kommander/1.x/kommander.yaml]: Please bump the revision for the addon, and then :+1:

Comment (DanielMSchmidt) [reply addons/kommander/1.x/kommander.yaml]: How do I update the addon revision?

Comment (shaneutt) [reply addons/kommander/1.x/kommander.yaml]: > catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.0-14"

becomes

> catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.0-15"

**Comment by DanielMSchmidt** [comment]
@shaneutt done 👍

**Comment by shaneutt** [comment]
master took the revision we wanted, so I bumped it again should be good to go now :+1:

**Comment by shaneutt** [comment]
@DanielMSchmidt it looks like there are some issues in CI here that may need resolving. The failures would seem to indicate that the configuration change made here causes Kommander to be unable to deploy properly. Please let @mesosphere/sig-ksphere-catalog know if you need any assistance.

**Comment by DanielMSchmidt** [comment]
CI looks green now, Can we merge this?

**Comment by jaxesn** [comment]
@DanielMSchmidt for the ticket I think that triggered this, we also want to remove, you can do that in another PR if you want, just want to make sure its not forgotten.  cc @jimmidyson https://github.com/mesosphere/kubeaddons-kommander/pull/18/files#diff-50426bf6792d43705c7848808dee0883R57

**Comment by shaneutt** [comment]
@DanielMSchmidt the revision got bumped ahead of us in master again, so I updated that.

**Comment by juliangieseke** [comment]
@jaxesn lets handle the KCL change in next PR, there is at least one more Bump necessary

**Pull request merged**